src/main/resources/templates/delete.ftl

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<body>
+<h3>Blog deletion</h3>
+<a href="http://localhost:8080/">Home</a>
+<a href="http://localhost:8080/blog/search/">Search blog</a>
```

Comment:
it'd be better to add a bit more semantic and use "nav", "ol/ul" and "li" tags as here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

The idea is that here you have a list of links, and with those links, you navigate through pages on your website